aio/Readme.md

```diff
@@ -0,0 +1,151 @@
+
+
+1. Install dependencies:
+
+        sudo apt-get update
+        sudo apt-get install memcached sqlite3 xfsprogs git-core
+
+1. Make sure Memcache is running:
+
+        sudo service memcached start
+
+Prerequisites: Go 1.8
+---------------------
+
+
+1. [Download Go binaries for your platform](https://golang.org/doc/install).
+
+1. Now, export three environment variables to inform both Go and your shell
+   where to find what you'll need to run both Go and Hummingbird binaries.
+
+   * `GOROOT` -- The `go` subdirectory of the location where your Go system
+     binaries and libraries are, usually `/usr/local/go`. This is not the
+     location of any code you write.
+
+   * `GOPATH` -- The folder where you intend to keep Go code, binaries, etc.
+     For simplicity, we'll do this in your home directory, so we'll use
+     `~/go`.
+
+   * `PATH` -- Append the bin dirs within both of the above onto your PATH,
+     with your normal path first, then $GOROOT/bin, and your $GOPATH/bin:
+
+          export GOROOT=/usr/local/go
+          export GOPATH=~/go
+          export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+
+1. It's also recommended to add these three lines either to the end of the
+   `/etc/profile` on the system, or at least to your own user's profile.
+
+1. Ensure you can now use the new `go` binary:
+
+        $ go version
+        go version go1.8.1 linux/amd64
+
+Installing Hummingbird
+----------------------
+
+1.  Cretate a directory for the Hummingbird code in GOPATH.  It's easiest to
+symlink it into a folder structure within GOPATH:
+
+        mkdir -p $GOPATH/src/github.com/troubling
+        ln -s ~/hummingbird $GOPATH/src/github.com/troubling/hummingbird
```

Comment:
This will create a directory called "hummingbird" in $GOPATH/src/github.com/troubling/hummingbird which will cause an error when you run:  
cd $GOPATH/src/github.com/troubling
git clone git@github.com:troubling/hummingbird.git

because the "hummingbird" directory already exists